### PR TITLE
Aave V3 Polygon MaticX Supply Cap and AGD Approval

### PR DESCRIPTION
**Pull request by efecarranza**
# Changelog

Add AIP text with supply cap increase and AGD approval update.

### CODE REVIEW

File: content/aips/AAVE-V3-POLYGON-CAPS-UPDATES-20230503.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+created: 2023-05-03
+---
+
+# Summary
+
+This publication proposes increasing the MaticX Supply Cap on Polygon v3 from 17.2M units to 29.3M units. It also corrects the Aave Grants DAO aUSDT allowance that was incorrectly set previously.
+
+# Abstract
+
+Aftera vote for increasing the communities counter party risk was passsed on Snapshot, the Supply Cap for MaticX can be increased from 50% to 75% of circulating supply.
```
Comment (sendra) [inline content/aips/AAVE-V3-POLYGON-CAPS-UPDATES-20230503.md:15]:
```suggestion
After a vote for increasing the communities counter party risk was passed on Snapshot, the Supply Cap for MaticX can be increased from 50% to 75% of circulating supply.
```

File: content/aips/AAVE-V3-POLYGON-CAPS-UPDATES-20230503.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+---
+title: MaticX Supply Cap Increase Polygon v3
```
Comment (sendra) [inline content/aips/AAVE-V3-POLYGON-CAPS-UPDATES-20230503.md:2]: Title and short description should also have AGD update

File: content/aips/AAVE-V3-POLYGON-CAPS-UPDATES-20230503.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+The current Supply Cap's utilisation is hovering around 94%, with 1.03M units of MaticX deposit capacity. Stader Labs is currently offering SD rewards to user who deposit MaticX. This proposal seeks to encourage Stader Labs to continue distributing SD rewards by increasing the Supply Cap and enabling more MaticX to be deposited into Aave. 
+
+# Motivation
+
+Over the previous months, Llama has been working with various communities to craft favourable conditions conducive to growing TVL and Revenue by facilitate the creation of several yield aggregation products. 
+
+With liquidity mining ongoing, MaticX deposit rewards (SD) and wMATIC borrowing rewards (stMATIC & MaticX), ensuring there is adequate capacity enabling the yield maximising strategies to grow is key for Aave Protcol's abililty to generate wMATIC revenue. Several liquidity pools on Balancer v2 now include bb-a-wMATIC with over 14.5M units of wMATIC being deposited in Aave Protocol. This has lead to reduced borrowing rates for wMATIC and supports the continued growth of yield maximising strategies on Aave Protocol. 
+
+![](https://i.imgur.com/CdVGNQ9.png)
```
Comment (sendra) [inline content/aips/AAVE-V3-POLYGON-CAPS-UPDATES-20230503.md:27]: image should be added on assets folder

**Comment by sendra** [comment]
Name of AIP file and title / short description should be updated to include AGD update

File: content/aips/AAVE-V3-POLYGON-CAPS-UPDATES-AGD-APPROVAL-20230503.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
+---
+title: MaticX Supply Cap Increase Polygon v3
```
Comment (sendra) [inline content/aips/AAVE-V3-POLYGON-CAPS-UPDATES-AGD-APPROVAL-20230503.md:2]: Missing AGD update

File: content/aips/AAVE-V3-POLYGON-CAPS-UPDATES-AGD-APPROVAL-20230503.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+discussions: https://governance.aave.com/t/arfc-maticx-supply-cap-increase-polygon-v3/12657
+shortDescription: Increase Supply Cap MaticX on Polygon v3 to 29.3M units.
```
Comment (sendra) [inline content/aips/AAVE-V3-POLYGON-CAPS-UPDATES-AGD-APPROVAL-20230503.md:4]: missing AGD update

**Review by sendra — APPROVED** [review]

**Review by eboadom — APPROVED** [review]

**Review by eboadom — APPROVED** [review]

**Pull request merged**